cmd/login/cmd.go

```diff
@@ -111,7 +112,7 @@ func init() {
 	flags.StringVar(
 		&args.env,
 		"env",
-		sdk.DefaultURL,
+		"",
```

Comment:
We can not remove this default, as right now its not needed and we default to prod, we should maintain this not to break backward compatibility

Reply:
its not removed, rather moved to pkg/ocm/config.go ResolveGatewayURL. there is still a default and for backwards compatability purposes its still `sdk.DefaultURL` (slightly oversimplified). We need the default cli arg to be "" so we can tell if they actually supplied the flag or not. 

Analyze `ResolveGatewayURL` if you want to double check me.